packages/edit-site/src/components/editor/index.js

```diff
@@ -237,6 +248,7 @@ export default function EditSiteEditor( { isPostsList = false } ) {
 					}
 					customSavePanel={ _isPreviewingTheme && <SavePanel /> }
 					forceDisableBlockTools={ ! hasDefaultEditorCanvasView }
+					forceRemoveBlockTools={ isGlobalStylesPreview }
```

Comment:
I'm not entirely sure this is needed either, I think we can probably remove all of these header tools when `isPreviewMode` is true.